ckanext/iati_generator/actions/iat_action.py

```diff
@@ -0,0 +1,97 @@
+import logging
```

Comment:
Please rename this file just to ´iati.py´

Reply:
https://github.com/okfn/ckanext-iati-generator/pull/7/commits/e6d1682ac3ef24c29f9b0447af1d3be63ec13e94